internal/ui/composer_test.go

```
@@ -0,0 +1,336 @@
+	}
+
+	m := buildModel(t)
+	m.openQuickMode()
+	m.quick.input.SetValue("see ")
+	m.quick.input.CursorEnd()
+	m.quick.lastImageID = 1
+	m.quick.attachments = []imageAttachment{{id: 1}}
+	m.quick.input.InsertString(imageToken(1))
+
+	updated, cmd := m.Update(pasteImageMsg{target: composerQuick, id: 1, noImage: true})
+	m = updated.(*Model)
```

golangci-lint (2.12.2)

[error] type assertion must be checked

(forcetypeassert)